js/apps.js

```diff
@@ -24,4 +25,129 @@ menuList.forEach((el) => {
       el.style.backgroundColor = 'transparent';
     });
   });
-});
\ No newline at end of file
+});
+
+const projectDetails = [
```

Comment:
- [ ] KIindly style the popup menu well

Reply:
Hi @Kossi-stack, Please I don't understand this commit so far because it fits the Figma template design or is near it though...

Reply:
Please I don't understand your comment so far... Please consider highlighting the message. because I follow the standard and for the PR message it's just a branch that will be deleted later on... @Kossi-stack need it Please

Reply:
Could you fix this linter error you just added

Reply:
Okay, sir @ishimwezachee... I pushed it already